### ISSUE TITLE
expandDocStrings property is not working in cluecumber-maven

### ISSUE THREAD

**Issue by duke2048** [trivago/cluecumber#308]
**Describe the bug**
expandDocStrings property is not working in cluecumber-maven

**To Reproduce**
1. Go to your pom.xml and set this the property to true: `<expandDocStrings>true</expandDocStrings>`
2. Run the plugin `mvn cluecumber:reporting`
3. Check the logs: 
```
14:14:53.321 INFO: - fail pending/undefined scenarios : false
14:14:53.321 INFO: - expand before/after hooks        : false
14:14:53.321 INFO: - expand step hooks                : false
14:14:53.321 INFO: - expand doc strings               : false
14:14:53.321 INFO: - expand attachments               : false
```
5. See error: Open the report, the doc strings are not expanded.

**Expected behavior**
The doc strings should be expanded in the report

**Comment by bischoffdev** [comment]
Thanks for reporting, will check it!

**Comment by bischoffdev** [comment]
Fixed in 3.2.1 (deploying now, should be available on Maven Central today)
